examples/langchain_tool_usage.py

```diff
@@ -33,7 +33,7 @@ def main():
     # We need to also import WikipediaAPIWrapper
     # The map is a mapping of the module name to the attribute name
     # langchain_community.utilities.WikipediaAPIWrapper
-    wikipedia_query_tool = Tool.from_langchain(
+    wikipedia_query_tool = ToolCreate.from_langchain(
```

Comment:
why not just pass the langchain tool to `client.create_tool(...)`? I feel like using `ToolCreate` directly from the Python interface in this way is a bit weird.

Reply:
Suggested API: 
```
tool = client.load_tool_from_langchain(langchain_tool, additional_imports_module_attr_map={"langchain_community.utilities": "WikipediaAPIWrapper"}
```

Reply:
Ack